2016-2017_Starstruck/CrossBot Competition/src/CrossDriver.c

```diff
@@ -0,0 +1,302 @@
+/*
+ * SwerveLib.c
+ *
+ *  Created on: Jul 24, 2016
+ *      Author: Mikel Matticoli
+ */
+
+#include <main.h>
+
+ServoSystem drive;
+
+void driveInit() {
+	drive = servoInit(RP, R, false, PID_MOTOR_SCALE, PID_THRESH);
+	delay(1000);
+	printf("RET Pot: %p\t%d\n\r",drive.targetValue, *drive.targetValue);
```

Comment:
are these prints just for testing purposes or do we need to see these values on the LCD too?